packages/forms-web-app/src/routes/appellant-submission/submission-form.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const { question, save, remove } = require('../../dynamic-forms/controller');
+const validate = require('../../dynamic-forms/validator/validator');
+const {
+	validationErrorHandler
+} = require('../../dynamic-forms/validator/validation-error-handler');
+const dynamicReqFilesToReqBodyFiles = require('../../dynamic-forms/middleware/dynamic-req-files-to-req-body-files');
+const getJourneyResponse = require('../../dynamic-forms/middleware/get-journey-response-for-appellant');
+
+const router = express.Router();
+
+// question
+router.get('/appeals/householder/:question', getJourneyResponse, question);
+
+// save
+router.post(
+	'/appeals/householder/:question',
+	getJourneyResponse,
+	dynamicReqFilesToReqBodyFiles(),
+	validate(),
+	validationErrorHandler,
+	save
+);
+
+// remove answer - only available for some question types
+router.get('/appeals/householder/:question/:answerId', getJourneyResponse, remove);
```

Comment:
Unsure about what the exact routes should be, went with how the urls in the tickets look. Not sure if :sections would also be used here like in the lpa questionnaire

Reply:
Having been through this at length with Abby, Bryn and Steve the URLs are gunna be /appeals/:appealType/:section/:question. The submissionId is going to come in as a query param. Example: /appeals/householder/your-details/who-made-application?id={guid}

Reply:
it strikes me that since I lost the argument on the submissionId not being a query param that we'll need new controllers